pinot-controller/src/main/resources/Readme.md

```diff
@@ -0,0 +1,16 @@
+# How to setup Pinot UI for development 
+
+1. Make sure pinot backend is running on port 9000. Follow [this guide](https://github.com/apache/pinot?tab=readme-ov-file#building-pinot) for the same.
+2. Navigate to ui source code folder 
+```shell
+cd pinot-controller/src/main/resources
+```
+3. Install Required Packages. Make sure you are using node v14 or more specifially v14.18.1
+```shell
+npm install 
+```
+4. Start the Development Server
```

Comment:
probably could add a step 5 to go to localhost:8080 for completeness, but otherwise this lgtm. this is effectively what I do except I have to include some obtuse node ssl option